src/test/java/edu/hm/hafner/analysis/parser/HadoLintParserTest.java

```diff
@@ -0,0 +1,63 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.nio.file.FileSystems;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.AbstractParserTest;
+import edu.hm.hafner.analysis.FileReaderFactory;
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.ParsingException;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+import edu.hm.hafner.analysis.assertions.SoftAssertions;
+
+import static edu.hm.hafner.analysis.assertions.Assertions.*;
+import static org.assertj.core.api.Assertions.*;
+
+class HadoLintParserTest extends AbstractParserTest {
+
+    HadoLintParserTest() {
+        super("hadolint.json");
+    }
+
+    @Override
+    protected void assertThatIssuesArePresent(final Report report, final SoftAssertions softly) {
+        assertThat(report).hasSize(5);
+        assertThat(report.get(1)).hasFileName("Dockerfile");
```

Comment:
It would make sense to check for all properties of this issue. Or is there a specific reason why you did not?

Reply:
Thanks for the feedback. No specific reason will add more asserts here.
```suggestion
        softly.assertThat(report.get(0))
                .hasSeverity(Severity.WARNING_LOW);
        softly.assertThat(report.get(1))
                .hasLineStart(15)
                .hasCategory("DL3008")
                .hasMessage("Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`")
                .hasColumnStart(1)
                .hasFileName("Dockerfile")
                .hasSeverity(Severity.WARNING_HIGH);
        softly.assertThat(report.get(2))
                .hasSeverity(Severity.WARNING_NORMAL);
```